partner_firstname/__manifest__.py

```diff
@@ -18,7 +18,7 @@
     "maintainer": "Camptocamp, Acsone",
```

Comment:
This is not correct. The correct key is `maintainers` and value should be a list of github logins.

Reply:
Change also the version number if you want your script running.

Reply:
I thought this is done automatically with the correct merge command? But I agree the version number needs to be 18.0.1.2.0 after the merge. I can change this.

Regarding maintainer or maintainers, it is mixed everywhere and not used very often. Is there any documentation beside this where it is without the "s": https://www.odoo.com/documentation/18.0/developer/reference/backend/module.html
I tend to remove it totally.